src/components/Buttons/FileButton/__tests__/FileButton.spec.tsx

```diff
@@ -5,6 +5,11 @@ import userEvent from '@testing-library/user-event';
 import { FileButton } from '../FileButton';
 
 describe('<FileButton> that renders a button', () => {
+    it('should render nothing if no children are provided', () => {
+        const { container } = render(<FileButton>{}</FileButton>);
+
+        expect(container.firstChild).toBeNull();
```

Comment:
jest provides toBeEmptyDOMElement that might be more self explanatory to use in this type of cases.

Reply:
good one. I was looking for it, but somehow didn't find it... Thanks